manifests/a/AllroundAutomations/PLSQLDeveloper/15.0.3.2058/AllroundAutomations.PLSQLDeveloper.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: AllroundAutomations.PLSQLDeveloper
+PackageVersion: 15.0.3.2058
```

```suggestion
PackageVersion: 15.0.3
```
Version is marketed as `15.0.3`. We should use the actual DisplayVersion inside `AppsAndFeaturesEntries`